Jenkinsfile_CNP

```diff
@@ -11,6 +11,23 @@ def type = "java"
 def product = "ia"
 def component = "bail-case-api"
 
+def secrets = [
+  'ia-${env}': [
+    secret('idam-client-id', 'IA_IDAM_CLIENT_ID'),
+    secret('idam-secret', 'IA_IDAM_SECRET'),
```

Comment:
Are we using same secret keys as being used in Appeals? Do we need to confirm with someone on this?

Reply:
that's a good point, we need to confirm, let me find out

Reply:
confirmed with Rajesh that it's ok to proceed for now